code/__HELPERS/qdel.dm

```diff
@@ -1,7 +1,7 @@
 #define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE)
 #define QDEL_IN_CLIENT_TIME(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
 #define QDEL_NULL(item) qdel(item); item = null
-#define QDEL_LIST(L) if(L) { for(var/I in L) qdel(I); L.Cut(); }
+#define QDEL_LIST(L) if(L) { for(var/I in L) qdel(I); if(L) { L.Cut(); } }
```

Comment:
```suggestion
#define QDEL_LIST(L) if(L) { for(var/I in L) qdel(I); if(L) { L.Cut(); } }
```
Just do L?.Cut, its much cleaner, don't get rid of the lazylist one for now

Reply:
Alright